types/obs-store.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'obs-store';
```

Comment:
Would you mind running `yarn build` locally and checking through `packages/keyring-controller/dist` to ensure that an import for `obs-store` doesn't show up there?

Reply:
I can't find any import in `packages/keyring-controller/dist`, but `obs-store` is used by `@metamask/eth-keyring-controller`

Reply:
Ah, sorry, this was an outdated comment that I forgot to remove before submitting the review. This is fine!

Reply:
See note above — `obs-store` is already a TypeScript project so we shouldn't need to do this.